GameFiles/_integration.gsc

```diff
@@ -12,6 +12,7 @@ init()
     level.eventBus.failKey      = "fail";
     level.eventBus.timeoutKey   = "timeout";
     level.eventBus.timeout      = 30;
+    level.eventBus.gamename     = getDvar("gamename"); // We want to do a few small detail different on IW5 compared to IW4, nothing where 2 files would make sense.
```

Comment:
nit: for consistency add surrounding space "padding" for any value inside of parenthesis as that's the standard I opted for.
eg: `getDvar( "gamename" );`